bin/codemods/src/single-tree-rendering.js

```diff
@@ -35,14 +43,30 @@ export default function transformer( file, api ) {
 	}
 
 	/**
+	 * Check if `parameters` has `param` either as a string or as a name of
+	 * an object, which could be e.g. an `Identifier`.
+	 *
+	 * @param {array} parameters Parameters to look from
+	 * @param {string} parameter Parameter name
+	 * @returns {boolean}
+	 */
+	function hasParam( parameters, parameter ) {
+		return (
+			_.findIndex( parameters || [], parameter => {
+				( parameter.name || parameter ) === 'context';
+			} ) !== -1
```

Comment:
`_.findIndex() !== -1` should be synonymous to [`_.some()`](https://lodash.com/docs/4.17.4#some) which is a bit more semantic :slightly_smiling_face: